app/actions/plugins/index.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Client4} from '@mm-redux/client';
+import {resetToChannel} from '@actions/navigation';
+
+const RESET_TO_CHANNEL = 'RESET_TO_CHANNEL';
+
+export async function doPluginAction(pluginId, trigger, body) {
+    const response = await Client4.executePluginTrigger(pluginId, trigger, body);
+    if (response.action === RESET_TO_CHANNEL) {
+        resetToChannel();
```

Comment:
Why do we need to reset the stack?

Reply:
We don't. I didn't know how to go back to the channel from here, and that navigation function seemed to do the trick.
We have to decide eventually what "actions" exist, and what to do on each action. Be it "go back", "go to channel x", or anything like that.

Long term idea: "open the modal X from plugin Y"